### ISSUE TITLE
"Too many lyrics?"

### ISSUE THREAD

**Issue by Raemon** [gwillen/solstice-audio-test#193]
While testing this with a friend the other day, they tried to paste in some song lyrics and it said "too many lyrics." Not sure if that was an artificial limit for screen-size management or an actual limit but it wasn't a particularly long song and seems like there should be some way to do longer songs.

**Comment by jeffkaufman** [comment]
Can you include the lyrics here, so I can test?

**Comment by jeffkaufman** [comment]
(The current limit is because we are still returning metadata in a header, but if that is not working for reasonable songs I could move it into the response body)

**Comment by Raemon** [comment]
I can't remember the song unfortunately.  I just did a quick test and found if you add 4 stanzas to 5000 years it breaks:

A possible child,
Dreaming through the longest night,
A possible smile,
Waking to a distant light,

A possible world,
Tell me what you wanna see,
Where's that child going, tell me
Who's that child gonna be...

In five thousand years,
Whatcha want to do, whatcha wanna see?
In five thousand years
Where we want to go, who we want to be?

In five thousand years,
If we boldly set our sights,
Journey through the coldest night
In five thousand years...

Oh...

Five thousand years...

We could build a brand new home,
We could raise the glass domes high.
And in a century or three
Our children might look at the sky

And then at last they'd see
The distant yellow sun.
The cradle of humanity,
And all the things we might become...

In five thousand years
Whatcha want to do, whatcha wanna see, in another
Five thousand years
Where we want to go, who we want to be, in another

Five thousand years
If we sailed across the stars.
Unimaginably far
In five thousand years...

Oh...

Five thousand years

And maybe good folk still might die,
But maybe not, we gotta try
I don't quite know what shape we'll take
I don't quite know what world we'll make

I don't quite know how things might change
I don't quite know what rules we'd break
Our present selves might think it strange
But there's so many lives at stake...

And entropy is bearin' down
But we got tricks for sticking 'round.
And if we live to see the day
That yellow might fade to red then grey,

We'll take a moment, one by one
Turn to face the dying su-un
Bittersweetly wave goodbye--
The journey's only just begun...

In five thousand years
Whatcha want to do, whatcha wanna see, in another
Five million years
Where we want to go, who we want to be, in another

Five billion years
When all that we once knew is gone
Our legacy could carry on
In five billion years...

Five billion years
Whatcha want to do, whatcha want to see, in another
Five billion years
Where we wanna go, who we wanna be, in another

Five billion years
Where we wanna go?
Five billion years
Who we wanna be?

Five billion years
Where we wanna go?
Five billion years
Who we wanna be?

And maybe good folk still might die,
But maybe not, we gotta try
I don't quite know what shape we'll take
I don't quite know what world we'll make

I don't quite know how things might change
I don't quite know what rules we'd break
Our present selves might think it strange
But there's so many lives at stake...

And maybe good folk still might die,
But maybe not, we gotta try
I don't quite know what shape we'll take
I don't quite know what world we'll make

I don't quite know how things might change
I don't quite know what rules we'd break
Our present selves might think it strange
But there's so many lives at stake...
